webapp/src/components/App/Singlestudy/NavHeader/index.tsx

```diff
@@ -0,0 +1,339 @@
+      }}
+    >
+      <Box
+        sx={{
+          width: 1,
+          display: "flex",
+          flexDirection: "row",
+          justifyContent: "flex-start",
+          alignItems: "center",
+        }}
+      >
+        <ArrowBackIcon
+          color="secondary"
+          onClick={handleClickBack}
+          sx={{ cursor: "pointer" }}
+        />
+        <Button variant="text" color="secondary" onClick={handleClickBack}>
+          <Tooltip
+            title={isExplorer ? study?.name : t("global.studies")}
+            followCursor
+          >
+            <Typography variant="button">
+              {isExplorer ? t("button.back") : t("global.studies")}
+            </Typography>
+          </Tooltip>
+        </Button>
+      </Box>
+      <Box
+        sx={{
+          width: 1,
+          display: "flex",
+        }}
+      >
+        <NavHeaderCommands
+          study={study}
+          onOpenCommands={openCommands}
+          isExplorer={isExplorer}
+          onCopyId={handleCopyId}
+          onLaunch={handleLaunch}
+          onUnarchive={handleUnarchive}
+        />
+        <Button
+          size="small"
+          aria-controls="menu-study"
+          aria-haspopup="true"
+          id="menu-study"
+          variant="outlined"
+          color="primary"
+          sx={{ width: "auto", minWidth: 0, px: 0, marginLeft: 2 }}
+          onClick={handleClick}
+        >
+          <MoreVertIcon />
+        </Button>
+        <NavHeaderMenu
+          anchorEl={anchorEl}
+          openMenu={openMenu}
+          onClose={handleClose}
+          menuItems={menuItems}
+        />
+      </Box>
+      <NavHeaderInfo
+        study={study}
+        parent={parent}
+        childrenTree={childrenTree}
+      />
+      {study && openLauncherDialog && (
+        <LauncherDialog
+          open={openLauncherDialog}
+          studyIds={[study.id]}
+          onClose={() => setOpenLauncherDialog(false)}
+        />
+      )}
+      {study && openPropertiesDialog && (
+        <PropertiesDialog
+          open={openPropertiesDialog}
+          onClose={() => setOpenPropertiesDialog(false)}
+          study={study}
+          updateStudyData={updateStudyData}
+        />
+      )}
+      {openDeleteDialog && (
+        <ConfirmationDialog
+          title={t("dialog.title.confirmation")}
+          onCancel={() => setOpenDeleteDialog(false)}
+          onConfirm={handleDelete}
+          alert="warning"
+          open
+        >
+          <Box
+            sx={{
+              display: "flex",
+              flexDirection: "column",
+              alignItems: "flex-start",
+              width: 1,
+              height: "auto",
+              gap: 1,
+            }}
+          >
+            <Typography>{t("studies.question.delete")}</Typography>
+            <CheckBoxFE
+              value={deleteChildren}
+              label={t("studies.deleteSubvariants")}
+              onChange={(_e, checked) => setDeleteChildren(checked)}
```

Comment:
`_` or `e`

Reply:
old code didn't catch we refactored, my bad